src/components/EthPriceCard.tsx

```diff
@@ -123,7 +137,7 @@ const EthPriceCard = ({ className, isLeftAlign }) => {
     price = <Translation id="loading-error-refresh" />
   }
 
-  const isNegativeChange = state.percentChangeUSD && state.percentChangeUSD < 0
+  const isNegativeChange = state.percentChangeUSD < 0
```

Comment:
```suggestion
  const isNegativeChange = state?.percentChangeUSD < 0
```
Just looking at the changes from before makes me think percentChange can be falsy, so just through the ?. operand here I think would make this equivalent to what was there before.

Reply:
Agree!